kratos/python/add_linear_solvers_to_python.cpp

```diff
@@ -178,6 +179,7 @@ void  AddLinearSolversToPython(pybind11::module& m)
     .def(py::init<LinearSolverType::Pointer>())
     .def(py::init<LinearSolverType::Pointer, bool >())
     .def(py::init<Parameters >())
+    .def("GetIterationsNumber",&ScalingSolverType::GetIterationsNumber)
```

Comment:
```suggestion
```

this is only needed once in the baseclass

Reply:
Scaling is derived from linear solver, for which GetIterationsNumber is not added to python (not needed).

Reply:
but you are adding it in this PR, so it should be done consistently btw C++ and Python

Reply:
I am not sure if this is a good idea. However, if technically this is preferred, I am OK.

Reply:
thx 
(this is how it is also done in other classes, works bcs of inheritance)

Reply:
it is:
https://github.com/KratosMultiphysics/Kratos/blob/fe7c9894198df58e16dc08f5a4005559740b335c/kratos/python/add_linear_solvers_to_python.cpp#L121-L128

Reply:
Sorry, I checked the wrong file.

Reply:
Done.